boards/ssrc/saluki/src/CMakeLists.txt

```diff
@@ -63,6 +64,7 @@ else()
 		led.c
 		toc.c
 		mpfs_emmcsd.c
+		mpfs_spinor.c
```

Comment:
This mpfs_spinor.c is not included into the commit

Reply:
oops, thanks. added the file